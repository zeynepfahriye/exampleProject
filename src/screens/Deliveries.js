import { View, Text, SafeAreaView, FlatList,Platform } from 'react-native'
import React from 'react'
import { Card, Title } from 'react-native-paper'
import Header from '../components/Header'

const Deliveries = ({navigation}) => {
    const deliveries = [1,2]

  return (
    <SafeAreaView style={{flex:1,backgroundColor:'#e0b0ff'}}>
      {Platform.OS === 'ios' &&
    <View style={{
        width: "100%",
        height: 100, // For all devices, even X, XS Max
        position: 'absolute',
        top: 0,
        left: 0,
        backgroundColor: "orange"}}
   />}
    <Header title={"My deliveries"} hasBackButton={true} navigation={navigation}/>
    <FlatList
    data={deliveries}
    keyExtractor={(item,index)=>`deliveries${index}`}
    renderItem={({item,index})=>
    <Card style={{margin:10,borderWidth:1,marginTop:30}}
    onPress={()=>navigation.navigate("Delivery")}
    >
        <Card.Cover
        source={require('../assets/map.jpg')}
        />
        <Card.Title
        titleStyle={{fontWeight:'bold'}}
        title="01/01/1111"
        subtitle="Delivery person name"
        right={()=>
        <Text style={{fontWeight:'bold',color:'purple',marginHorizontal:10}}>$30</Text>
        }
        >
        </Card.Title>
    </Card>
    }
    >
    </FlatList>
    
    </SafeAreaView>
  )
}

export default Deliveries