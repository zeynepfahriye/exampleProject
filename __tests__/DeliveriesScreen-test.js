import React from 'react'
import 'react-native'
import renderer from 'react-test-renderer'
import Deliveries from '../src/screens/Deliveries'

test('Form header snapshot',()=>{
    const snap = renderer.create(
        <Deliveries />
    ).toJSON();
    expect(snap).toMatchSnapshot();
})