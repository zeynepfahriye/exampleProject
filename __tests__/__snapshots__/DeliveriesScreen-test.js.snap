// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form header snapshot 1`] = `
<RCTSafeAreaView
  style={
    Object {
      "backgroundColor": "#e0b0ff",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "orange",
        "height": 100,
        "left": 0,
        "position": "absolute",
        "top": 0,
        "width": "100%",
      }
    }
  />
  <View
    collapsable={false}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "orange",
        "elevation": 4,
        "flexDirection": "row",
        "height": 56,
        "paddingHorizontal": 4,
        "shadowColor": "#000000",
        "shadowOffset": Object {
          "height": 3,
          "width": 0,
        },
        "shadowOpacity": 0.24,
        "shadowRadius": 4,
      }
    }
  >
    <View
      accessibilityLabel="Back"
      accessibilityRole="button"
      accessibilityState={
        Object {
          "disabled": false,
        }
      }
      accessible={true}
      focusable={true}
      hitSlop={
        Object {
          "bottom": 6,
          "left": 6,
          "right": 6,
          "top": 6,
        }
      }
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "overflow": "hidden",
          },
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
              "margin": 6,
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 18,
              "height": 36,
              "width": 36,
            },
            undefined,
            undefined,
          ],
        ]
      }
    >
      <View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "justifyContent": "center",
              },
              Object {
                "height": 24,
                "width": 24,
              },
            ]
          }
        >
          <View
            collapsable={false}
            style={
              Object {
                "bottom": 0,
                "left": 0,
                "opacity": 1,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "transform": Array [
                  Object {
                    "rotate": "0deg",
                  },
                ],
              }
            }
          >
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "justifyContent": "center",
                  },
                  Object {
                    "height": 24,
                    "transform": Array [
                      Object {
                        "scaleX": 1,
                      },
                    ],
                    "width": 24,
                  },
                ]
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../react-native-paper/lib/commonjs/assets/back-chevron.png",
                  }
                }
                style={
                  Array [
                    Object {
                      "resizeMode": "contain",
                    },
                    Object {
                      "height": 21,
                      "tintColor": "white",
                      "width": 21,
                    },
                  ]
                }
              />
            </View>
          </View>
        </View>
      </View>
    </View>
    <View
      accessibilityState={
        Object {
          "disabled": true,
        }
      }
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "flex": 1,
            "paddingHorizontal": 12,
          },
          Array [
            Object {
              "marginLeft": 8,
            },
            Object {
              "alignItems": "center",
            },
            undefined,
          ],
        ]
      }
    >
      <Text
        accessibilityRole="header"
        accessibilityTraits="header"
        accessible={true}
        numberOfLines={1}
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Object {
              "textAlign": "left",
            },
            Array [
              Object {
                "color": "white",
                "fontFamily": "System",
                "fontWeight": "400",
              },
              Object {
                "fontSize": 17,
              },
              undefined,
            ],
          ]
        }
      >
        My deliveries
      </Text>
    </View>
    <View
      style={
        Object {
          "width": 48,
        }
      }
    />
  </View>
  <RCTScrollView
    data={
      Array [
        1,
        2,
      ]
    }
    getItem={[Function]}
    getItemCount={[Function]}
    keyExtractor={[Function]}
    onContentSizeChange={[Function]}
    onLayout={[Function]}
    onMomentumScrollBegin={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    viewabilityConfigCallbackPairs={Array []}
  >
    <View>
      <View
        onLayout={[Function]}
        style={null}
      >
        <View
          collapsable={false}
          style={
            Object {
              "backgroundColor": "#ffffff",
              "borderColor": "rgba(0, 0, 0, 0.12)",
              "borderRadius": 4,
              "borderWidth": 1,
              "elevation": 1,
              "margin": 10,
              "marginTop": 30,
              "shadowColor": "#000000",
              "shadowOffset": Object {
                "height": 0.5,
                "width": 0,
              },
              "shadowOpacity": 0.24,
              "shadowRadius": 0.75,
            }
          }
        >
          <View
            accessibilityState={
              Object {
                "disabled": false,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "flexGrow": 1,
                "flexShrink": 1,
              }
            }
          >
            <View
              style={
                Array [
                  Object {
                    "backgroundColor": "#eeeeee",
                    "height": 195,
                    "overflow": "hidden",
                  },
                  Object {
                    "borderTopLeftRadius": 4,
                    "borderTopRightRadius": 4,
                  },
                  undefined,
                ]
              }
            >
              <Image
                siblings={
                  Array [
                    "withTheme(Card.Cover)",
                    "withTheme(Card.Title)",
                  ]
                }
                source={
                  Object {
                    "testUri": "../../../src/assets/map.jpg",
                  }
                }
                style={
                  Array [
                    Object {
                      "flex": 1,
                      "height": undefined,
                      "justifyContent": "flex-end",
                      "padding": 16,
                      "width": undefined,
                    },
                    Object {
                      "borderTopLeftRadius": 4,
                      "borderTopRightRadius": 4,
                    },
                  ]
                }
              />
            </View>
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": "space-between",
                    "paddingLeft": 16,
                  },
                  Object {
                    "minHeight": 72,
                  },
                  undefined,
                ]
              }
            >
              <View
                style={
                  Array [
                    Object {
                      "flex": 1,
                      "flexDirection": "column",
                      "justifyContent": "center",
                    },
                  ]
                }
              >
                <Text
                  numberOfLines={1}
                  style={
                    Array [
                      Object {
                        "color": "#000000",
                        "fontFamily": "System",
                        "fontWeight": "400",
                      },
                      Object {
                        "textAlign": "left",
                      },
                      Array [
                        Object {
                          "textAlign": "left",
                        },
                        Object {
                          "color": "rgba(0, 0, 0, 0.87)",
                          "fontFamily": "System",
                          "fontWeight": "500",
                          "writingDirection": "ltr",
                        },
                        Array [
                          Object {
                            "fontSize": 20,
                            "letterSpacing": 0.15,
                            "lineHeight": 30,
                            "marginVertical": 2,
                          },
                          Array [
                            Object {
                              "minHeight": 30,
                              "paddingRight": 16,
                            },
                            Object {
                              "marginBottom": 0,
                            },
                            Object {
                              "fontWeight": "bold",
                            },
                          ],
                        ],
                      ],
                    ]
                  }
                >
                  01/01/1111
                </Text>
                <Text
                  numberOfLines={1}
                  style={
                    Array [
                      Object {
                        "color": "#000000",
                        "fontFamily": "System",
                        "fontWeight": "400",
                      },
                      Object {
                        "textAlign": "left",
                      },
                      Array [
                        Object {
                          "textAlign": "left",
                        },
                        Object {
                          "color": "rgba(0, 0, 0, 0.54)",
                          "fontFamily": "System",
                          "fontWeight": "400",
                          "writingDirection": "ltr",
                        },
                        Array [
                          Object {
                            "fontSize": 12,
                            "letterSpacing": 0.4,
                            "lineHeight": 20,
                            "marginVertical": 2,
                          },
                          Array [
                            Object {
                              "marginVertical": 0,
                              "minHeight": 20,
                              "paddingRight": 16,
                            },
                            undefined,
                          ],
                        ],
                      ],
                    ]
                  }
                >
                  Delivery person name
                </Text>
              </View>
              <View>
                <Text
                  style={
                    Object {
                      "color": "purple",
                      "fontWeight": "bold",
                      "marginHorizontal": 10,
                    }
                  }
                >
                  $30
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
      <View
        onLayout={[Function]}
        style={null}
      >
        <View
          collapsable={false}
          style={
            Object {
              "backgroundColor": "#ffffff",
              "borderColor": "rgba(0, 0, 0, 0.12)",
              "borderRadius": 4,
              "borderWidth": 1,
              "elevation": 1,
              "margin": 10,
              "marginTop": 30,
              "shadowColor": "#000000",
              "shadowOffset": Object {
                "height": 0.5,
                "width": 0,
              },
              "shadowOpacity": 0.24,
              "shadowRadius": 0.75,
            }
          }
        >
          <View
            accessibilityState={
              Object {
                "disabled": false,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "flexGrow": 1,
                "flexShrink": 1,
              }
            }
          >
            <View
              style={
                Array [
                  Object {
                    "backgroundColor": "#eeeeee",
                    "height": 195,
                    "overflow": "hidden",
                  },
                  Object {
                    "borderTopLeftRadius": 4,
                    "borderTopRightRadius": 4,
                  },
                  undefined,
                ]
              }
            >
              <Image
                siblings={
                  Array [
                    "withTheme(Card.Cover)",
                    "withTheme(Card.Title)",
                  ]
                }
                source={
                  Object {
                    "testUri": "../../../src/assets/map.jpg",
                  }
                }
                style={
                  Array [
                    Object {
                      "flex": 1,
                      "height": undefined,
                      "justifyContent": "flex-end",
                      "padding": 16,
                      "width": undefined,
                    },
                    Object {
                      "borderTopLeftRadius": 4,
                      "borderTopRightRadius": 4,
                    },
                  ]
                }
              />
            </View>
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": "space-between",
                    "paddingLeft": 16,
                  },
                  Object {
                    "minHeight": 72,
                  },
                  undefined,
                ]
              }
            >
              <View
                style={
                  Array [
                    Object {
                      "flex": 1,
                      "flexDirection": "column",
                      "justifyContent": "center",
                    },
                  ]
                }
              >
                <Text
                  numberOfLines={1}
                  style={
                    Array [
                      Object {
                        "color": "#000000",
                        "fontFamily": "System",
                        "fontWeight": "400",
                      },
                      Object {
                        "textAlign": "left",
                      },
                      Array [
                        Object {
                          "textAlign": "left",
                        },
                        Object {
                          "color": "rgba(0, 0, 0, 0.87)",
                          "fontFamily": "System",
                          "fontWeight": "500",
                          "writingDirection": "ltr",
                        },
                        Array [
                          Object {
                            "fontSize": 20,
                            "letterSpacing": 0.15,
                            "lineHeight": 30,
                            "marginVertical": 2,
                          },
                          Array [
                            Object {
                              "minHeight": 30,
                              "paddingRight": 16,
                            },
                            Object {
                              "marginBottom": 0,
                            },
                            Object {
                              "fontWeight": "bold",
                            },
                          ],
                        ],
                      ],
                    ]
                  }
                >
                  01/01/1111
                </Text>
                <Text
                  numberOfLines={1}
                  style={
                    Array [
                      Object {
                        "color": "#000000",
                        "fontFamily": "System",
                        "fontWeight": "400",
                      },
                      Object {
                        "textAlign": "left",
                      },
                      Array [
                        Object {
                          "textAlign": "left",
                        },
                        Object {
                          "color": "rgba(0, 0, 0, 0.54)",
                          "fontFamily": "System",
                          "fontWeight": "400",
                          "writingDirection": "ltr",
                        },
                        Array [
                          Object {
                            "fontSize": 12,
                            "letterSpacing": 0.4,
                            "lineHeight": 20,
                            "marginVertical": 2,
                          },
                          Array [
                            Object {
                              "marginVertical": 0,
                              "minHeight": 20,
                              "paddingRight": 16,
                            },
                            undefined,
                          ],
                        ],
                      ],
                    ]
                  }
                >
                  Delivery person name
                </Text>
              </View>
              <View>
                <Text
                  style={
                    Object {
                      "color": "purple",
                      "fontWeight": "bold",
                      "marginHorizontal": 10,
                    }
                  }
                >
                  $30
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</RCTSafeAreaView>
`;
